src/include/daos/dtx.h

```diff
@@ -43,17 +43,7 @@ struct dtx_conflict_entry {
 	uint64_t		dce_dkey;
 };
 
-static inline void
-daos_dti_gen(struct dtx_id *dti, bool zero)
-{
-	if (zero) {
-		memset(dti, 0, sizeof(*dti));
-	} else {
-		/* It will be replaced by HLC when it is ready. */
-		uuid_generate(dti->dti_uuid);
-		dti->dti_sec = time(NULL);
```

Comment:
Next step, we can remove the dti_sec since the dti_uuid always contains HLC in the patch.

Reply:
the hlc in the dti_uuid is only temporary used to generate a pseudo-uuid, probably still need dti_sec for its usage.
BTW, I changed the gethostid() as I found got same value on different boro nodes.